client/hudwidget.cpp

```diff
@@ -1834,17 +1837,24 @@ void hud_unit_combat::paintEvent(QPaintEvent *event)
   p.setFont(f);
   p.setPen(QColor(Qt::white));
   if (def_veteran) {
-    p.drawText(right,
-               Qt::AlignHCenter | Qt::AlignJustify | Qt::AlignAbsolute,
-               QStringLiteral("*"));
+    p.drawText(right, Qt::AlignBottom | Qt::AlignRight | Qt::AlignAbsolute,
+               _("*Vet*"));
```

Comment:
What are the stars for? Maybe change the color to something goldy and leave a hint for translators?

Reply:
As you can see, the original `drawText()` simply showed a star for a vet, which wasn't clear. I can look at an alternate color or simply change to remove the stars.